src/Monitor.vala

```diff
@@ -28,7 +52,21 @@ namespace Monitor {
         }
 
         public static int main (string [] args) {
-            var app = new MonitorApp ();
+
```

Comment:
Please no new line after declaring a function. See https://elementary.io/docs/code/reference#whitespace for more details